dotcom-rendering/src/model/enhance-dividers.test.ts

```diff
@@ -1,4 +1,4 @@
-import { Standard as ExampleArticle } from '../../fixtures/generated/articles/Standard';
+import { Standard as ExampleArticle } from '../../fixtures/generated/dcr-articles/Standard';
```

Comment:
Would it be correct to say the enhancers essentially do `FEArticleType -> DCRArticle`? If so, does that mean these enhancer test files should be importing `fe-articles`?